test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SchedulesExpiryDuringReconnect.java

```diff
@@ -85,7 +85,8 @@ private HapiApiSpec expireSchedulesDuringReconnect() {
 								.hasAnyKnownStatus()
 								.deferStatusResolution()
 								.adminKey(DEFAULT_PAYER)
-								.logging().advertisingCreation()
+								.logging()
```

Comment:
So this `.logging()` and the next 2 are still not meant to be `.nologging()`?

Reply:
This class creates only 3 schedules, so I added logging. Other two classes create 300k tokens, and 20k schedules. So I removed logging to reduce log size.